mne_bids_pipeline/steps/preprocessing/_03_maxfilter.py

```diff
@@ -378,6 +378,12 @@ def run_maxwell_filter(
         head_pos=head_pos,
         extended_proj=extended_proj,
     )
+    if duplicates := (set(cfg.mf_extra_kws) & set(mf_kws)):
+        raise RuntimeError(
```

Comment:
is there a better error class than `RuntimeError` that I should be using instead?  ConfigError or similar?

Reply:
ValueError perhaps?

Reply:
ValueError perhaps?

Reply:
Agreed. Would it be worth adding a `mne_bids_pipeline.ConfigError` subclass?
```
class ConfigError(ValueError):
    pass
```

Also this validation would ideally happen in with the rest of the config validation stuff in `_config_import.py` but I understand if it's a pain because we'd have to list the set of (dis)allowed `kwargs` over there probably, which isn't elegant

Reply:
it wasn't hard. see [647b830](https://github.com/mne-tools/mne-bids-pipeline/pull/1038/commits/647b830c453a8c2d6d7ec1d6c11eadabc4834155)